src/lib/chips/_chips-theme.scss

```diff
@@ -34,12 +30,21 @@ $mat-chip-remove-font-size: 18px;
   $unselected-background: mat-color($background, unselected-chip);
   $unselected-foreground: mat-color($foreground, text);
 
-
   .mat-chip.mat-standard-chip {
     @include mat-chips-color($unselected-foreground, $unselected-background);
 
-    &:focus {
-      @include _mat-theme-elevation(3, $theme);
+    &:not(.mat-chip-disabled) {
+      &:focus, &:active {
```

Comment:
Looking at the spec again, focused chips aren't supposed to have an elevation, but rather a darker overlay:

![spec](https://storage.googleapis.com/spec-host-backup/mio-design%2Fassets%2F1SdXde429CX7iphzp_SawbCRBZmKZ4q8d%2Finput-chips-states.png)

Reply:
Yeah, I can change that while being at it. Should I also make the overlay darker if focused?

Reply:
Sure.